### PR TITLE
chore: allow to load from custom URL

### PR DESCRIPTION
**Pull request by obenjiro**
Fix for https://github.com/huggingface/ratchet/issues/208

### CODE REVIEW

File: crates/ratchet-web/src/model.rs [hunk 1]
```diff
@@ -344,4 +381,43 @@ mod tests {
...
+    #[wasm_bindgen_test]
+    async fn whisper_browser_custom() -> Result<(), JsValue> {
+        log_init();
+        let download_cb: Closure<dyn Fn(f64)> = Closure::new(|p| {
+            log::info!("Provided closure got progress: {}", p);
+        });
+        let js_cb: &js_sys::Function = download_cb.as_ref().unchecked_ref();
+
+        let mut model = Model::load_custom(
+            "https://huggingface.co/FL33TW00D-HF/whisper-base2/resolve/main".to_string(),
```
Comment (FL33TW00D) [inline crates/ratchet-web/src/model.rs:394]: Is this going to work?

Comment (obenjiro) [reply crates/ratchet-web/src/model.rs:394]: Yep. That dose change base URL + path. So if you upload sane files to different location everything is going to work 

Checked that with wasm_bindgen_test

Comment (FL33TW00D) [reply crates/ratchet-web/src/model.rs:394]: No no all good! Sorry couldn't test as there is an issue with upstream, working on it now.

Comment (obenjiro) [reply crates/ratchet-web/src/model.rs:394]: Is there anything I can do to help?

Comment (FL33TW00D) [reply crates/ratchet-web/src/model.rs:394]: No it's upstream, don't worry about it i will disable CI for now.

File: crates/ratchet-web/src/model.rs [hunk 2]
```diff
@@ -203,6 +203,43 @@ impl Model {
         })
     }
 
+    #[wasm_bindgen]
+    pub async fn load_custom(
```
Comment (FL33TW00D) [inline crates/ratchet-web/src/model.rs:207]: This method and load are 99% the same.

Can't we extract the core logic into `load_inner`, and then expose `load_custom` and `load`?

Comment (obenjiro) [reply crates/ratchet-web/src/model.rs:207]: Will do

Comment (FL33TW00D) [reply crates/ratchet-web/src/model.rs:187]: Nice!

**Pull request merged**